### ISSUE TITLE
Alpha channel not working properly using `hex8` mode in Nova 3 (v `1.2`)

### ISSUE THREAD

**Issue by reppair** [outl1ne/nova-color-field#12]
Hi there.

I am having issues using the alpha channel in either `hex8` or `rgba` mode using version `1.2.2` of the package. Not sure if the problem is also present in v2 and Nova 4.

### HEX8 mode (#rrggbbAA notation)

Using `Color::make('hex8')->chrome()->displayAs('hex8')->saveAs('hex8')` - picked example color `#6657aa94`

- **will** properly store the picked color with the alpha channel to the database - `#6657aa94`
- **won't** properly render the color on details page using the hex string - it uses `background-color: rgba(102, 87, 170, 0.58);` instead of `style="background-color: #6657aa94;`
- **won't** properly load the alpha to the color picker when editing a record to change the color

| Actual | Expected |
| ------ | --------- |
| <img width="360" alt="Screen Shot 2022-09-15 at 15 20 48" src="https://user-images.githubusercontent.com/10784954/190402273-2dac42fa-9c97-40b9-a7cd-a4261de406c8.png"> |  <img width="312" alt="Screen Shot 2022-09-15 at 15 22 32" src="https://user-images.githubusercontent.com/10784954/190402441-711eb01e-fe2a-47d2-8794-ca1fa2608b5a.png"> |

Note it would be nice if the main color (the square) also is updated with the proper color after the picker is closed (or even while the color picker is open).

## Related issues
- #2

Let me know what do you think. I could work on those fixes and enhancements this weekend maybe.

**Comment by KasparRosin** [comment]
These seem like valid issues. Pull request is more than welcome.

**Comment by reppair** [comment]
Will be able to look into it this week.

**Comment by reppair** [comment]
📅 Busy days but still on my list.

**Comment by reppair** [comment]
The rendering of the color as `rgba` regardless of passing `hex(8)` to it seems to be out of scope here.

See: https://github.com/vuejs/core/issues/2998

**Comment by reppair** [comment]
Updated the issue to reference only the alpha problem when using `hex8` and got a PR for it here #15 (for `v1` - Nova 3).

Will create a separate issue for the alpha issue when using `rgba` since this will be harder to fix. The issue there is a bit more complicated and will require more time to fix and refactor.
Maybe can give that a try when I got some free time but for now I am happy with `hex8` working properly. 😺
